sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml

```diff
@@ -118,7 +119,7 @@ extensions:
                          --inventory sjb/inventory/ \
                          -e deployment_type=origin  \
                          -e openshift_use_crio=True   \
-                         -e openshift_crio_systemcontainer_image_override=docker.io/gscrivano/cri-o-centos \
+                         -e openshift_crio_systemcontainer_image_override="docker.io/runcom/cri-o-system-container:v${crio_tag}" \
```

Comment:
Is ``crio_tag`` being set? In output that @mrunalp passed over to @sdodson and I the end result looked like:

```docker.io/runcom/cri-o-system-container:v```

Reply:
true.  If this is the version currently used by the CI then `crio_tag` looks empty

Reply:
This *seems* to be merged .... but not "merged". 😕

Reply:
I just talked with @runcom He deployed it to test to see if it works. We need to debug why the tag isn't getting set.

Reply:
yup, I'll debug it asap

Reply:
@runcom were you able to find the issue?

Reply:
I'm debugging here just right now, I'll update this once I find something

Reply:
should be fixed now, I'll work on Steve's switch case for images

Reply:
I've deployed this to the origin CI already so we can test if it fixes the empty crio_tag issue